torch/ao/quantization/fx/_model_report/model_report.py

```diff
@@ -6,12 +6,24 @@
     DETECTOR_OBS_ARGS_KEY,
     DETECTOR_OBS_TO_INSERT_KEY,
     DETECTOR_IS_POST_OBS_KEY,
-    DETECTOR_TARGET_NODE_KEY
+    DETECTOR_TARGET_NODE_KEY,
+    DynamicStaticDetector,
+    PerChannelDetector
 )
 from torch.ao.quantization.fx._model_report.model_report_visualizer import ModelReportVisualizer
 from torch.ao.quantization.fx.graph_module import GraphModule
 from torch.ao.quantization.observer import ObserverBase
-from torch.ao.quantization.qconfig_mapping import QConfigMapping
+from torch.ao.quantization.qconfig_mapping import QConfigMapping, QConfig
+from torch.ao.quantization.qconfig import (
+    default_qconfig,
+    assert_valid_qconfig,
+)
+from torch.ao.quantization.observer import (
+    default_dynamic_quant_observer,
+    default_per_channel_weight_observer,
+    default_observer,
+    default_weight_observer,
+)
 
 class ModelReport:
```

Comment:
I think ideally, each detector will be able to generate a qconfig_mapping and the job for model_reporter class is to merge them in some way, this will make the code more modular, is this possible?